components/lib/hooks/index.ts

```diff
@@ -0,0 +1,14 @@
+export { default as useWalletAddress } from "./useWalletAddress";
+export { default as useWeb3Provider } from "./useWeb3Provider";
+export { default as useSigner } from "./useSigner";
+export { default as useDeployedAddress } from "./useDeployedAddress";
+export { default as useTransactionLogger } from "./useTransactionLogger";
+export type { Transaction } from "./useTransactionLogger";
+export { default as useBalances } from "./useBalances";
+export type { Balances } from "./useBalances";
+export { default as useEffectAsync } from "./useEffectAsync";
+export { default as useWeb3 } from "./useWeb3";
+
+export { default as useDeployedContracts } from "./contracts/useDeployedContracts";
+export { default as useNamedContracts } from "./contracts/useNamedContracts";
+export { default as useManagerManaged } from "./contracts/useManagerManaged";
```

Comment:
Strange syntax; you should be able to import all these like as follows

```typescript
export useManagerManaged from "./contracts/useManagerManaged";
```

lol